### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by tehnewb** [crashinvaders/gdx-texture-packer-gui#114]
> Please paste the crash log in here (already in your clipboard) and, if possible, provide any information that may help to reproduce the issue.
<details><summary>Application Log</summary>

```
Version: 4.10.0
OS: Windows 10 10.0 amd64
JRE: 16.0.2 Oracle Corporation
GPU: Intel(R) UHD Graphics 630
OpenGL vendor: Intel
OpenGL version: 4.6.0 - Build 27.20.100.8190
[ExtensionModuleRepositoryService] Cached data was loaded
[ConfigurationController] Incremental fonts are disabled
[VisUI] Warning, using invalid libGDX version for VisUI 1.4.8.
You are using libGDX 1.9.11 but you need 1.9.12. This may cause unexpected problems and runtime exceptions.
[GlobalShortcutHandler] Parsing shortcut file: hotkeys_default.txt
Packing is started
Packing.......
Writing 24x24: C:\Users\Amy\Desktop\CharacterPack.png
Packing is done
Total pack files size is 1471 bytes
1 pages
Total processing time: 0.06sec

Packing is started
Packing.......
Writing 24x24: C:\Users\Amy\Desktop\CharacterPack.png
Packing is done
Total pack files size is 1468 bytes
1 pages
Total processing time: 0.03sec

java.lang.RuntimeException: Actor: T
	at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:192)
	at com.badlogic.gdx.scenes.scene2d.Actor.fire(Actor.java:152)
	at com.badlogic.gdx.scenes.scene2d.Stage.keyTyped(Stage.java:446)
	at com.crashinvaders.common.PrioritizedInputMultiplexer$Wrapper.keyTyped(PrioritizedInputMultiplexer.java:128)
	at com.crashinvaders.common.PrioritizedInputMultiplexer.keyTyped(PrioritizedInputMultiplexer.java:65)
	at com.badlogic.gdx.InputEventQueue.drain(InputEventQueue.java:81)
	at com.badlogic.gdx.backends.lwjgl3.DefaultLwjgl3Input.update(DefaultLwjgl3Input.java:187)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:394)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:141)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:117)
	at com.crashinvaders.texturepackergui.desktop.ApplicationStarter.start(ApplicationStarter.java:56)
	at com.crashinvaders.texturepackergui.desktop.ApplicationStarter.main(ApplicationStarter.java:35)
Caused by: java.lang.RuntimeException: Actor: T
	at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:192)
	at com.badlogic.gdx.scenes.scene2d.Actor.fire(Actor.java:152)
	at com.kotcrab.vis.ui.widget.VisTextField.changeText(VisTextField.java:702)
	at com.kotcrab.vis.ui.widget.VisTextField$TextFieldClickListener.keyTyped(VisTextField.java:1232)
	at com.badlogic.gdx.scenes.scene2d.InputListener.handle(InputListener.java:57)
	at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:188)
	... 11 more
Caused by: java.lang.IllegalArgumentException: actor cannot be null.
	at com.badlogic.gdx.scenes.scene2d.ui.Table.getCell(Table.java:417)
	at com.kotcrab.vis.ui.widget.MenuItem.getImageCell(MenuItem.java:352)
	at com.kotcrab.vis.ui.widget.file.internal.AbstractSuggestionPopup.createMenuItem(AbstractSuggestionPopup.java:36)
	at com.kotcrab.vis.ui.widget.file.internal.FileSuggestionPopup.createSuggestions(FileSuggestionPopup.java:55)
	at com.kotcrab.vis.ui.widget.file.internal.FileSuggestionPopup.pathFieldKeyTyped(FileSuggestionPopup.java:41)
	at com.kotcrab.vis.ui.widget.file.FileChooser$16.changed(FileChooser.java:558)
	at com.badlogic.gdx.scenes.scene2d.utils.ChangeListener.handle(ChangeListener.java:28)
	at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:188)
	... 16 more
```
</details>

**Comment by metaphore** [comment]
Thanks for the report!
I believe this happened in the result of typing a path inside File Chooser dialog? If so the crash is fixed for #98

Please feel free to reopen if the actual context is wrong and needs an investigation.

**Comment by tehnewb** [comment]
> Thanks for the report!
> I believe this happened in the result of typing a path inside File Chooser dialog? If so the crash is fixed for #98
> 
> Please feel free to reopen if the actual context is wrong and needs an investigation.

I've re-downloaded the releases a couple times, but still get this error. If it was fixed, then how do I find the fixed version?

**Comment by metaphore** [comment]
It's not released yet.
Is that bug that critical and there's no workaround to select a dir for you?
